plugins/inputs/bond/bond.go

```diff
@@ -0,0 +1,172 @@
+package bond
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+// default bond directory path
+const (
+	BOND_PATH = "/proc/net/bonding"
```

Comment:
I would say call this `defaultBondPath` because Go naming guidelines, but to match existing style we should only specify the proc location.  We can add it to the config as `host_proc = "/proc"`, but we should also load it from the `HOST_PROC` environment variable.

Reply:
done